src/main/java/com/cburch/logisim/file/XmlReader.java

```diff
@@ -428,6 +428,7 @@ private void toLogisimFile(Element elt, Project proj) {
         isEvolutionFile = true;
         OptionPane.showMessageDialog(
             null,
+            // FIXME: none langualized string
```

Comment:
can you please make simple `hardcoded string` comment instead? It took me a while to understand what I just read + it will be easier to find it later if it is tagged same way as other similar strings.

Reply:
Done